packages/3cpio/project.bri

```diff
@@ -0,0 +1,43 @@
+import * as std from "std";
+import { cargoBuild } from "rust";
+
+export const project = {
+  name: "3cpio",
+  version: "0.12.0",
+  extra: {
+    crateName: "threecpio",
+  },
+};
+
+const source = Brioche.download(
+  `https://crates.io/api/v1/crates/${project.extra.crateName}/${project.version}/download`,
+)
+  .unarchive("tar", "gzip")
+  .peel();
+
+export default function _3cpio(): std.Recipe<std.Directory> {
```

Comment:
Before to merge this one, should we name the main function `_3cpio` or `threecpio` ? As a result, should the name of the recipe follow the name of the function if we go for `treecpio` ?

Interestingly, Nix choose a different approach here, they named the package [`_3cpio`](https://github.com/NixOS/nixpkgs/blob/nixos-25.11/pkgs/by-name/_3/_3cpio/package.nix).

Reply:
I like the way it's set up here the best. The README shows that the project itself is called "3cpio", and I like sticking to the official project name as much as possible (Nix I think probably needed to call it something else for similar reasons why the crate had to be called `threecpio` in crates.io, due to identifier rules or something)

For the local function name here, I kinda lean towards `_3cpio` as well-- just seems simpler to add an underscore in front as a general convention. But I honestly don't feel very strongly about it, since this name is purely local to this file